src/app/components/site/Header.tsx

```diff
@@ -4,12 +4,10 @@ import { selectors, useAppSelector } from "../../state";
 import { Col, Container, Row } from "reactstrap";
 import { MainSearch } from "../elements/MainSearch";
 import { NavigationBar } from "./NavigationBar";
-import { useDeviceSize } from "../../services";
 
 export const Header = () => {
```

Comment:
likewise with the search icon in it's opened and closed state, and the search bar button when expanded please: 

<img width="378" alt="image" src="https://github.com/user-attachments/assets/8331f208-454d-4427-8b0b-011f91fa0db4">

Reply:
I have made the box slightly larger as per design. The svgs inside align in the middle. 
I have aligned the search button in centre position to be parallel to the search bar.